classes/View.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class View
+{
+    private $template;
+    
+    public function __construct($template, $frontback)
+    {
+        $this->template = $template;
+        $this->frontback = $frontback;
+    }
+    
+    public function render($params = array())
+    {
+        //On récupère dans le controller, le fait que c'est du backend ou du frontend
+        $frontback = $this->frontback;
+        
+        // Appel à l'UserManager nécessaire pour la topbar_backend
+        $manager_user = new UserManager();
+        $exist = $manager_user->admin();
+       
+        // Affichage de la topbar
+        $topbar="views/topbar_$frontback.php";
+        
+        extract($params); // extrac() parcourt le tableau de $params et créé posts, responses, post, etc.
+        
+        //Affichage du contenu
+        $template = $this->template;
+        
+        ob_start(); //On met dans une mémoire tampon.
+
+        include("views/$frontback/$template.php");
+        $contentPage = ob_get_clean();
+
+        include("views/layout.php");        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["include" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)